core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -574,21 +580,50 @@ object PSBT extends Factory[PSBT] {
 
   def fromUnsignedTxAndInputs(
       unsignedTx: Transaction,
-      spendingInfos: Vector[UTXOSpendingInfo])(
+      spendingInfoAndNonWitnessTxs: Vector[
```

Comment:
can we get a type for this `Vector[(UTXOSPendingInfo, Option[Transaction])]` or at least a type for the tuple inside?